acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/AcceptanceTestBase.java

```diff
@@ -28,6 +28,8 @@
 
 public class AcceptanceTestBase {
 
+  public static final String RECIPIENT = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
```

Comment:
Don't have this variable here in the base class, as it's not useful to **all** the AT extending it i.e it's not a common field.

Reply:
done